src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -58,11 +64,28 @@ const props = defineProps({
     type: String,
     default: 'Clear filters',
   },
+  areFiltersHidden: {
+    type: Boolean,
+    default: () => undefined,
+  },
+  hiddenFiltersNames: {
+    type: Array,
+    default: () => [],
+  },
```

Comment:
`hiddenFilterNames` enough to define what filter to hide(if filter is in the array - it is hidden).
areFiltersHidden is not necessary, but it could be replaced by prop `isClearButtonHidden`

So you have full control what to show and what to hide.

Reply:
Can we place the 'isHidden' in the actual `filters` object array, then we control the filter and can toggle its visibility from the parent component. The filters object looks like: 
```
    label: ...,
    name: ....,
    component: ....,
    items: [....],
    isHidden: boolean
```

Ditto on the button prop @wolandec suggested,

Reply:
I'm agree with both changes, I'm going to change them